spec/spec_helper.rb

```diff
@@ -1,5 +1,7 @@
-require 'coveralls'
-Coveralls.wear!
+unless ENV['SKIP_COVERALLS']
```

Comment:
Added to skip the following error with newer Ruby versions on CI

```
[Coveralls] Submitting to https://coveralls.io/api/v1
/Users/runner/hostedtoolcache/Ruby/2.7.8/x64/lib/ruby/gems/2.7.0/gems/json-1.8.6/lib/json/common.rb:155: warning: Using the last argument as keyword parameters is deprecated
```


As well as not having a coveralls token setup in Github actions, and potentially the coveralls API being changed since [coveralls ~> 0.8](https://github.com/packetfu/packetfu/blob/31cbecca9cd6d3486629ff4797433784a37ce2e4/packetfu.gemspec#L25)

```
Coveralls encountered an exception:
JSON::ParserError
822: unexpected token at '<html>
<head><title>400 Bad Request</title></head>
<body>
<center><h1>400 Bad Request</h1></center>
<hr><center>nginx</center>
</body>
</html>
'
```